lutris/util/wine/proton.py

```diff
@@ -40,7 +40,7 @@ def is_umu_path(wine_path: Optional[str]) -> bool:
     PROTONPATH, but if this is omitted it will default to the latest Proton it
     downloads."""
     if wine_path:
-        return wine_path.endswith("/umu_run.py") or wine_path.endswith("/umu-run")
+        return wine_path.endswith("/umu-run")
```

Comment:
I don't like that you are dropping support for the .py entry points- this means older Umu runtimes we've had won't work.

Reply:
If Lutris decides to subscribe to using a standalone executable, this problem can resolved by simply creating a relative umu_run.py symbolic link to umu-run in its build process

Reply:
No, that's not enough. We've already shipped an Umu runtime- we should not break it until we've actually replaced it. It has no 'umu-run' in it, just 'umu_run.py'.